### DISCUSSION TITLE
BlueSCSI works fine with Power Macintosh 6500/275

### DISCUSSION THREAD

**Discussion by MrGasS** [erichelgeson/BlueSCSI-v1#99] [Show and tell]
Today I tested my handmade BlueSCSI (based on a real Blue Pill, on firmware 1.1-20220404) on my Power Macintosh 6500/275, everything works fine and the Blue Pill powers up without issues from the external DB-25 port. I installed 8.5 on it and boots :)

![IMG_20220423_155643952](https://user-images.githubusercontent.com/37547183/164909537-463f423b-7700-434f-a18d-1935bf4858b3.jpg)
![IMG_20220423_154654313](https://user-images.githubusercontent.com/37547183/164909539-294ec64b-988b-418a-967a-79e5b348dba0.jpg)
![IMG_20220423_155518536](https://user-images.githubusercontent.com/37547183/164909544-6c719ee1-53c4-406b-a551-fdd9cc1b981a.jpg)
![IMG_20220423_155535928](https://user-images.githubusercontent.com/37547183/164909548-5f9616cf-02f1-4990-af5e-57baa8a21553.jpg)
![IMG_20220423_155601143](https://user-images.githubusercontent.com/37547183/164909551-a5f3bae4-b7bf-4011-86d5-8965c4a73052.jpg)

**Comment by Elv1sP** [comment]
I got a problem on my 6500/250, I use the top space in stead of the zipdrive with the berg connector and it freezes the computer randomly (mouse stops and computer is unresponsive) sometimes gives crash bombs like error 41 and others.

**Reply by MrGasS** [reply]
did you try another SD card?

**Reply by Elv1sP** [reply]
Yes, bought new a new one as well. Tryed SanDisk and now latest Kingston. Get fragmentet files. So I ordered a new SD card again today, this time from Verbatim

**Reply by MrGasS** [reply]
Did you solve this problem?
I'm sorry, but I'm not able to do more tests anymore because I sold my 6500.

**Comment by Elv1sP** [comment]
Hi, I have not yet solved the problem.  MVHRobin Westelius 1 maj 2023 kl. 13:00 skrev EnricoGasS ***@***.***>:﻿
Did you solve this problem?
I'm sorry, but I'm not able to do more tests anymore because I sold my 6500.

**Comment by Elv1sP** [comment]
I get crazy, why cant I get my bluescsi v1.1 to work with the  toolbox cd changer. Says ”No compatable bluescsi found” but I run my drive from it and shared folder is working.




It is update to latest version, but I cant update with the USB version for some reason, my bluepill has the usb port on it
MVH
Robin Westelius

**Reply by erichelgeson** [reply]
BlueSCSI v1 only supports the Transfer app. CD Changer only works on a BlueSCSI v2.

**Comment by Elv1sP** [comment]
Ok thnx for Quick answer. I must have missed that bit of informationMVHRobin Westelius 13 dec. 2023 kl. 17:22 skrev Eric Helgeson ***@***.***>:﻿
BlueSCSI v1 only supports the Transfer app. CD Changer only works on a BlueSCSI v2.
